components/fxa-client/ios/FxAClient/FxAccount.swift

```diff
@@ -116,25 +116,25 @@ class FxAccount: RustFxAccount {
         }
     }
 
-    override func migrateFromSessionToken(sessionToken: String, kSync: String, kXCS: String) -> Bool {
+    override func migrateFromSessionToken(sessionToken: String, kSync: String, kXCS: String) -> [MigrationData]? {
```

Comment:
Are we returning multiple `MigrationData` objects? This could be simplified to a non-array.

Reply:
Yeap, that works!